bin/split.sh

```diff
@@ -18,60 +18,14 @@ function remote()
 
 git pull origin $CURRENT_BRANCH
 
-remote auth git@github.com:illuminate/auth.git
-remote broadcasting git@github.com:illuminate/broadcasting.git
-remote bus git@github.com:illuminate/bus.git
-remote cache git@github.com:illuminate/cache.git
-remote config git@github.com:illuminate/config.git
-remote console git@github.com:illuminate/console.git
-remote container git@github.com:illuminate/container.git
-remote contracts git@github.com:illuminate/contracts.git
-remote cookie git@github.com:illuminate/cookie.git
-remote database git@github.com:illuminate/database.git
-remote encryption git@github.com:illuminate/encryption.git
-remote events git@github.com:illuminate/events.git
-remote filesystem git@github.com:illuminate/filesystem.git
-remote hashing git@github.com:illuminate/hashing.git
-remote http git@github.com:illuminate/http.git
-remote log git@github.com:illuminate/log.git
-remote mail git@github.com:illuminate/mail.git
-remote notifications git@github.com:illuminate/notifications.git
-remote pagination git@github.com:illuminate/pagination.git
-remote pipeline git@github.com:illuminate/pipeline.git
-remote queue git@github.com:illuminate/queue.git
-remote redis git@github.com:illuminate/redis.git
-remote routing git@github.com:illuminate/routing.git
-remote session git@github.com:illuminate/session.git
-remote support git@github.com:illuminate/support.git
-remote translation git@github.com:illuminate/translation.git
-remote validation git@github.com:illuminate/validation.git
-remote view git@github.com:illuminate/view.git
+for REMOTE in Broadcasting Bus Cache Config Console Container Contracts Cookie Database Encryption Events Filesystem Hashing Http Log Mail Notifications Pagination Pipeline Queue Redis Routing Session Support Translation Validation View
```

Comment:
One thing about the "old" system I find more useful: you don't have to scroll a list from left to right >100 chars to check something: an alphabetical top-down list is much easier to scan, edit (diffs)

Reply:
Another option:

```sh
for REMOTE in \
    Broadcasting \
    Bus \
    Cache \
    Config \
    Console \
    ...
do
    echo $REMOTE
done
```

Reply:
See 4fd8b48555030e46be4171a53619a19731a020bd